language/diem-framework/modules/0L/Globals.move

```diff
@@ -146,7 +146,7 @@ module Globals {
           // target transaction per sec max gas: 20
           // uses "scaled representation", since there are no decimals.
           subsidy_ceiling_gas: 8640000 * COIN_SCALING_FACTOR, // subsidy amount assumes 24 hour epoch lengths. Also needs to be adjusted for coin_scale the onchain representation of human readable value.
-          vdf_difficulty_baseline: 120000000, // FYI approx 30 mins per proof on 2020 macbook pro 2.5 ghz quadcore
+          vdf_difficulty_baseline: 160000000, // FYI approx 30 mins per proof on 2020 macbook pro 2.5 ghz quadcore
```

Comment:
Prob this comment is no longer true, right?

Reply:
Good point. Will update once I get the final confirmation for the best value to set.